stack/indexer/indexer-security-init.sh

```diff
@@ -35,6 +35,24 @@ WAZUH_INDEXER_ROOT_CA="$(cat ${CONFIG_FILE} 2>&1 | grep http.pemtrustedcas | sed
 WAZUH_INDEXER_ADMIN_PATH="$(dirname "${WAZUH_INDEXER_ROOT_CA}" 2>&1)"
 SECURITY_PATH="${INSTALL_PATH}/plugins/opensearch-security"
 
+# -----------------------------------------------------------------------------
+
+trap ctrl_c INT
+
+clean(){
+
+    exit_code=$1
+    indexer_process_id=$(pgrep -f wazuh-indexer -c)
+    if [ "${indexer_process_id}" -gt 1 ]; then
+        pkill -n -f wazuh-indexer
```

Comment:
Why are you stopping the process with kill instead of systemctl?

Reply:
I don't want to stop wazuh-indexer process, just kill the newly created ones that contains that name when the script is executed. There is always a wazuh-indexer process running which contains opensearch and if we end it or stop it the script can't be launch again unless this process is restored.